src/pool/srv_pool.c

```diff
@@ -1810,6 +1810,11 @@ pool_svc_step_up_cb(struct ds_rsvc *rsvc)
 	} else {
 		uuid_generate(pool_hdl_uuid);
 		uuid_generate(cont_hdl_uuid);
+		/* Only copy server handle to make is_from_srv() check correctly, and
+		 * container server handle will not be copied here, otherwise
+		 * ds_pool_iv_refresh_hdl will not open the server handle container.
```

Comment:
[Nit] Does the first line mean "pool server handle" instead of "server handle"? On the last line, what is "the server handle container"?

Reply:
oh, pool server handle means pool handle for server to server communication.  Sorry, it should be container server handle.